docs/docs/deployment/docker.md

```diff
@@ -17,10 +17,8 @@ services:
         ports:
             - "3000:3000"
         environment:
-            NEXT_PUBLIC_API_BASE_URL: "https://bracket.mywebsite.com"
-            # Go to https://dashboard.hcaptcha.com/signup, create a site and put the site key here
-            NEXT_PUBLIC_HCAPTCHA_SITE_KEY: "xxxxx"
-            NODE_ENV: "production"
+NEXT_PUBLIC_API_BASE_URL: "http://your-site.com:8400"
+            NEXT_PUBLIC_HCAPTCHA_SITE_KEY: "10000000-ffff-ffff-ffff-000000000001"  # Public test key
```

Comment:
I would prefer to keep the instruction `# Go to https://dashboard.hcaptcha.com/signup, create a site and put the site key here` here, since it is more clear what the value here should be. Putting a test key here would probably result in users just copy/pasting and accidentally using the test key in production without noticing I think